Makefile

```diff
@@ -153,13 +156,13 @@ lint: lint-gosec
 	@./scripts/import-order-cleanup.sh stdout > $(IMPORT_LOG)
 	@[ ! -s "$(FMT_LOG)" -a ! -s "$(IMPORT_LOG)" ] || (echo "Go fmt, license check, or import ordering failures, run 'make fmt'" | cat - $(FMT_LOG) && false)
 
-.PHONY: install-glide
```

Comment:
Could we keep this for a while, as an alias to `install-dep` + WARN message stating that it's deprecated?

Reply:
Sure good idea.